src/contract.rs

```diff
@@ -137,9 +137,23 @@ pub fn execute(
                 return contract_error("only gp can redeem capital");
             }
 
-            let send = BankMsg::Send {
-                to_address: to.to_string(),
-                amount: coins(amount as u128, state.capital_denom),
+            let capital_marker =
+                ProvenanceQuerier::new(&deps.querier).get_marker_by_denom(&state.capital_denom)?;
+
+            let mut bank_send: Option<BankMsg> = None;
+            let mut marker_transfer: Option<CosmosMsg<ProvenanceMsg>> = None;
```

Comment:
I couldn't find a good way to combine a `BankMsg` with a `ProvenanceMsg` so I went with the two split out variables

Reply:
I believe there is a path to accomplish that, but even if not that specifically. 

One could also start the builder above, and add message to the builder within the branching logic here as well.

Reply:
That is a great suggestion.... I couldn't see the forest through the trees trying to figure out the other aspects of the language 😄